docs/audit_report/src/02_security_issues.rst

```diff
@@ -1,7 +1,27 @@
 Security and Vulnerabilities
 ============================
 
-Currently, we are not aware of any security issues that needed to be addressed
-between |botan_git_base_ref| and |botan_version|.
+.. _secinfo/kyberslash:
 
-.. todo:: Update this in case some vulnerability is found and fixed before the 3.3.0 release.
+KyberSlash
+----------
+
+This is a series of potential side channel vulnerabilities discovered in several
+software implementations of Kyber. It was initially discovered by Goutam
+Tamvada, Franziskus Kiefer, and Karthikeyan Bhargavan and later `independently
+reported by Daniel Bernstein
+<https://groups.google.com/a/list.nist.gov/g/pqc-forum/c/hWqFJCucuj4/m/-Z-jm_k9AAAJ>`_.
+A `similar issue was discovered by Prasanna Ravi and Matthias Kannwischer
+<https://groups.google.com/a/list.nist.gov/g/pqc-forum/c/ldX0ThYJuBo/m/ovODsdY7AwAJ>`_
+in a different part of the implementation shortly after.
+
+The timing side channels stem from divisions by a constant integer (`q`) in the
+C++ code. On some target configurations (compiler, processor architecture, build
+flags) this division operation is represented by a variable-time division
+instruction, which leaks information.
+
+Affected are the serialization of Kyber's shared secret value during
+decapsulation as well as the serialization of the ciphertext during
+encapsulation. The latter is relevant because encapsulation is used for
```

Comment:
As far as I understand [this post](https://groups.google.com/a/list.nist.gov/g/pqc-forum/c/ldX0ThYJuBo/m/uIOqRF5BAwAJ), DJB was able to learn the entire secret key using only the KyberSlash1 side channel. Therefore, I think both side channels are relevant. As far as I know, the second side channel can be mainly used to break Kyber's _implicit rejection_ approach (i.e., if decryption fails, a dummy shared secret is outputted anyway). I don't know what evil mathematicians can or cannot do with this knowledge and some maliciously crafted chosen ciphertexts. But at first glance, it's not as troublesome as the first one.

Reply:
Fair point.